radio/src/pulses/modules_helpers.h

```diff
@@ -487,7 +487,7 @@ inline int8_t minModuleChannels(uint8_t idx)
   else if (isModuleSBUS(idx))
     return 16;
   else if (isModuleDSMP(idx))
-    return maxModuleChannels(idx);
+    return 1;
```

Comment:
"module assume always start with Ch1." 

Was this just lost in translation? As this shouldn't in relation to the starting channel, but the *minimum* number of channels, i.e. so you can't bring the change range fields any closer than this value (or any bigger than maxModuleChannels).